packages/kbn-i18n/GUIDELINE.md

```diff
@@ -333,3 +418,25 @@ it('should render normally', async () => {
 });
 // ...
 ```
+
+## Development steps
+
+1. Apply appropriate `i18n` component.
+
+2. Check UI that nothing is broken.
+
+3. Check functionality of an element (button is clicked, checkbox is checked/unchecked, etc.).
+
+4. Run validation tool:
+    ```js
+      node scripts/i18n_check --output ./
+    ```
+    and look through created `en.json`.
+
+5. Run eslint, tslint rules.
+
+6. Run tests.
+
+7. Check with pseudo-locale by setting `i18n.locale` to `en-xa` in `./config/kibana.yml`.
```

Comment:
__nit:__ Run Kibana with enabled pseudo-locale (either pass `--i18n.locale=en-xa` as a command-line argument or add it to the `kibana.yml`) and observe the text you've just localized. If you did everything correctly, it should turn into something like this `Ĥéļļļô ŴŴôŕļļð!` assuming your text was `Hello World!`.